Automattic/wp-calypso
# WooCommerce: Add tax settings view, including persisting to server

Fixes #14439 
Fixes #14440 
Fixes #14441 
Fixes #13483

To test:

- After setting the store address during setup, navigate to store/settings/taxes/{site}
- Make sure the address is presented correctly
- Make sure you can edit the address. Rates should update when you edit the address.
- Make sure the tax information presented for the state and country are correct
- Make sure you can toggle taxes enabled on and off and save the change.
- Make sure you can toggle check boxes.
- Run `npm run test-client client/extensions/woocommerce/state`
